modules/mobile/docs/schemas/LighthouseErrors.yml

```diff
@@ -0,0 +1,21 @@
+type: object
+required:
+  - errors
+properties:
+  data:
```

Comment:
Do you need this "data"? Doesn't this imply that there's a property called within each error?

Reply:
good catch. fixed